core-web/src/main/java/org/silverpeas/core/webapi/base/WebResourceProvider.java

```diff
@@ -35,4 +36,8 @@
 @ApplicationPath(SilverpeasWebResource.BASE_PATH)
 public class WebResourceProvider extends Application {
 
+  @Override
+  public Set<Class<?>> getClasses() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1185)